src/common/models/Projections.ts

```diff
@@ -15,37 +15,36 @@ import { RegionSummaryWithTimeseries } from 'api/schema/RegionSummaryWithTimeser
  * we're not focused on projections and there's only 1.
  */
 export class Projections {
-  stateCode: string;
-  stateName: string;
-  county: any;
-  countyName: string | null;
   primary: Projection;
   isCounty: boolean;
+  region: Region;
 
   constructor(
     summaryWithTimeseries: RegionSummaryWithTimeseries,
-    stateCode: string,
-    county?: any,
+    region: Region,
   ) {
-    this.stateCode = stateCode.toUpperCase();
-    this.stateName = (STATES as any)[this.stateCode];
-    this.county = null;
-    this.countyName = null;
-    this.isCounty = county != null;
+    this.region = region;
+    this.isCounty = region instanceof County;
     this.primary = new Projection(summaryWithTimeseries, {
       isCounty: this.isCounty,
     });
-
-    this.populateCounty(county);
   }
 
   populateCounty(county: any) {
     if (!county) {
       return;
     }
+  }
+
+  get fips(): string {
+    return this.primary.fips;
```

Comment:
Should we return `region.fipsCode` instead?

Reply:
I don't think it matters too much here, they should never be different